src/tx/applyTx/checkSpendCond.js

```diff
@@ -455,7 +461,10 @@ module.exports = async (state, tx, bridgeState, nodeConfig = {}) => {
         // this hack assumes that an ERC1949 is minted
         // and that Transfer Event is emmited before UpdateData Event
         // so in only puts the new owner into the nftBag
-        nftBag[originAddr][`0x${topics[3].toString('hex')}`] = toAddr;
+        nftBag[originAddr][`0x${topics[3].toString('hex')}`] = {
```

Comment:
Would be nice to extract `0x${topics[3].toString('hex')}` into variable:
1. The thing will have a name
2. It used more than 1 time